tools/publish.sh

```diff
@@ -33,13 +33,13 @@ crates=(
     bevy_dylib
 )
 
-cd crates
+cd crates || set -ex
```

Comment:
I actually mean `set -ex` on top of the file and then remove the `|| exit 1`. It should have been `set -eou pipefail` instead of `set -ex` it seems by the way. `set -e` causes the script to exit if any command exists with a non-zero exit code. `set -u` causes undefined variables to be treated as error. `set -o pipefail` causes pipelines (`|`) to exit with a non-zero exit code if any command that is part of it exits with a non-zero exit code.